pkg/universal-cnf/config/mocks/UniversalCNFBackend.go

```diff
@@ -0,0 +1,88 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
I am not really familiar with the nsm-nse project, but if this is a new tool to generate mocks in this project, consider adding some comments on how to use it. If there is any readme file or smtg.

Reply:
Thanks for the suggestion! I will add some comments on how to generate mocks